packages/cli/cloud/src/services/strapi-info-save.ts

```diff
@@ -1,20 +1,35 @@
 import fse from 'fs-extra';
 import path from 'path';
-import type { ProjectInfos } from './cli-api';
+import type { ProjectInfo } from './cli-api';
 
 export const LOCAL_SAVE_FILENAME = '.strapi-cloud.json';
 
 export type LocalSave = {
-  project?: Omit<ProjectInfos, 'id'>;
+  project?: Omit<ProjectInfo, 'id'>;
 };
 
-export async function save(data: LocalSave, { directoryPath }: { directoryPath?: string } = {}) {
+export async function addEnvironment(
+  targetEnvironment: string | undefined,
+  { directoryPath }: { directoryPath?: string } = {}
+) {
   const alreadyInFileData = await retrieve({ directoryPath });
-  const storedData = { ...alreadyInFileData, ...data };
+  if (!alreadyInFileData || !alreadyInFileData.project) {
+    throw new Error('No valid configuration found in the local configuration file.');
+  }
+  const dataToSave: LocalSave = {
+    ...alreadyInFileData,
+    project: {
+      ...alreadyInFileData.project,
+      targetEnvironment,
+    },
+  };
+  await save(dataToSave, { directoryPath });
+}
+
+export async function save(data: LocalSave, { directoryPath }: { directoryPath?: string } = {}) {
   const pathToFile = path.join(directoryPath || process.cwd(), LOCAL_SAVE_FILENAME);
-  // Ensure the directory exists
```

Comment:
You don't like my comments? 🥲
Maybe a more explicit one could be better like

// Ensure the directory exists and creates it if not.

As not everyone is knowing the functions use in fs-extra 🤔

Reply:
Re-added comment